apps/worker/src/app/workflow/usecases/execute-bridge-job/execute-bridge-job.usecase.ts

```diff
@@ -249,12 +249,23 @@ export class ExecuteBridgeJob {
       let raw: { retryCount?: number; statusCode?: number; message: string; url?: string };
 
       if (error.response) {
+        let rawMessage: Record<string, unknown>;
+        const errorResponseBody = error?.response?.body;
+        try {
+          rawMessage = JSON.parse(errorResponseBody);
+        } catch {
+          Logger.error(`Unexpected body received from Bridge: ${errorResponseBody}`, LOG_CONTEXT);
+          rawMessage = {
+            error: `Unexpected body received from Bridge: ${errorResponseBody}`,
+          };
+        }
+
         raw = {
           url: statelessBridgeUrl,
           statusCode: error.response?.statusCode,
           message: error.response?.statusMessage,
           ...(error.response?.retryCount ? { retryCount: error.response?.retryCount } : {}),
-          ...(error?.response?.body?.length > 0 ? { raw: JSON.parse(error?.response?.body) } : {}),
+          ...(error?.response?.body?.length > 0 ? { raw: rawMessage } : {}),
```

Comment:
I believe this lack of error handling was contributing to execution detail logs not surfacing due to an inability to parse the error body. This was observed with errors such as "BSON circular value cannot be converted..."